meilisearch/version.py

```diff
@@ -0,0 +1,6 @@
+VERSION = "0.18.0"
+
+def qualified_version() -> str:
```

Comment:
Yes, but I don't know if defining a function is the best way to achieve this, look: https://github.com/meilisearch/integration-guides/issues/150

The main point is to share the package version in this format: `"Meilisearch Python (v1.1.1)"` over the User-Agent header but I will do this in the next PR.

Reply:
Actually @brunoocasali thinking about this more, the way you plan to use the version, putting `__version__` into the `__init__.py` file will cause a circular import. Because of that I guess you should keep the `version.py` file, but I would still change the variable name to `__version__`.